### DISCUSSION TITLE
Animal Breakpoints

### DISCUSSION THREAD

**Discussion by HMueller0** [msberends/AMR#118] [Q&A] [answered]
Hello there,
i want to analyse MIC-data of animals and I think the AMR-package could be a great tool for that. 

However, I have read that the clinical_breakpoints-dataframe only contains human breakpoints and that interpretation guidelines from CLSI for animals were removed. I saw that it is possible to create custom-made breakpoints (by using the argument reference data and by creating a dataframe which has identical columns as clinical_breakpoints. I started creating a dataframe for animals manually, but I just wanted to ask you , if there is a more efficient and reliable way to use animal-breakpoints in the AMR-package?

Thanks a lot!

**Comment by msberends** [comment, marked as answer]
Hi! Thank you for reaching out. Perhaps it’s not too bad to include animal guidelines as well. In issue #102 we talked about ECOFF values, which was also considered important for veterinary medicine. 

The primary reason not to include them was because of package size, but we didn’t analyse this through and through. I can see in the next weeks what the impact would be regarding package size and maintenance. But it would surely enlarge the scope for the package.

**Reply by HMueller0** [reply]
Thanks for including animal guidelines. That's great!

**Reply by msberends** [reply]
You’re welcome!
